.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,16 @@
+### Description
+
+<!--
+Please explain the changes you made here.
+
+Help your reviewers my guiding them through your key changes,
+implementation decisions etc.
+You can even include snippets of output or screenshots.
+
+A good, clear description == a faster review :)
+-->
+
+### Checklist
+- [ ] Added tests that cover your change
+- [ ] Added/modified documentation as required (such as the `README.md`)
+- [ ] Made sure the title of the PR is a good description that can go into the release notes
```

Comment:
If we have the PR labels workflow, do we want to have the add labels check here?

Reply:
done